cli/runtime/config/cli_discovery_sources_test.go

```diff
@@ -268,6 +268,7 @@ func TestDeleteCLIDiscoverySource(t *testing.T) {
 		src     *configapi.ClientConfig
 		input   string
 		deleted bool
```

Comment:
nit: I don't see .deleted getting used anywhere

Reply:
not needed I have used errStr for the same. removed `deleted`